modules/margin/src/main/java/com/opengamma/sdk/margin/MarginCalcRequest.java

```diff
@@ -109,6 +116,39 @@ public static MarginCalcRequest of(
    * The list of portfolio data should be either {@link PortfolioDataFile} instances
    * or Strata {@code CalculationTarget} instances, such as trades or positions.
    *
+   * @param mode  the mode of the calculation
+   * @param valuationDate  the valuation date
+   * @param reportingCurrency  the reporting currency
+   * @param portfolioData  the portfolio data, which can be {@link PortfolioDataFile} or Strata trades
+   * @param fpmlPartySelectionRegex  the regular expression used to select the party in any FpML input
+   * @return the request
+   */
+  public static MarginCalcRequest of(
+      MarginCalcMode mode,
```

Comment:
This should be after the reporting currency - its not as important as date or currency